free-3.md

```diff
@@ -57,12 +114,27 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Even discounting the UI interface of this project, ther would still be a large
+amount of this project that would be semantics. The AI part of the language is
+rather difficult because it has to follow the rules of the game and hopefully
+be a competitive player to the point that it's moves are not self-destruction.
```

Comment:
If you modify the purpose of the DSL so that it's for game designers, you would remove the need to build an AI for the game while still preserving the need for a DSL that lets people input rules and cards to simulate a game for playtesting.